R4ML/R/sysml.bridge.R

```diff
@@ -50,6 +50,194 @@ sysml.MatrixCharacteristics <- setRefClass("sysml.MatrixCharacteristics",
 
 )
 
+#' @export
```

Comment:
@bdwyer2 see the jira it says WIP, and it especifically says that it is not yet ready